librz/bin/format/elf/elf.h

```diff
@@ -253,7 +231,12 @@ ut64 Elf_(rz_bin_elf_get_main_offset)(RZ_NONNULL ELFOBJ *bin);
 
 // elf_notes.c
 
-bool Elf_(rz_bin_elf_init_notes)(RZ_NONNULL ELFOBJ *bin);
+RZ_BORROW RzBinElfPrStatusLayout *Elf_(rz_bin_elf_get_prstatus_layout)(RZ_NONNULL ELFOBJ *bin);
+RZ_BORROW RzVector *Elf_(rz_bin_elf_get_notes_file)(RZ_NONNULL ELFOBJ *bin);
+RZ_BORROW RzVector *Elf_(rz_bin_elf_get_notes_prstatus)(RZ_NONNULL ELFOBJ *bin);
```

Comment:
do we really need a specific API just for prstatus? Maybe we can have a more generic API with `type` passed as argument, since there may be many other types of notes, right?

Reply:
Agree on that suggestion, especially since future GCC/Binutils might add more with time.

Reply:
I modified the API